src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -142,6 +157,15 @@ public function __construct(
         $this->fieldDescriptorFactory = $fieldDescriptorFactory;
         $this->mediaClass = $mediaClass;
         $this->collectionClass = $collectionClass;
+        $this->mediaListBuilderFactory = $mediaListBuilderFactory;
+        $this->mediaListRepresentationFactory = $mediaListRepresentationFactory;
+
+        if (null === $this->mediaListBuilderFactory) {
```

Comment:
Same as above, we probably have to do the same check for the other newly introduced arguments as well.